api/python/provisioner/commands/set_cluster_id.py

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com."
+#
+
+# Python API to set unique ClusterID to pillar file
```

Comment:
We also need the 4 steps to set cluster_id here involving check for srvnode-*.

Reply:
1. Check if cluster-id is set in grains using grains_get API
2. Check if cluster-id is set in pillar using pillar_get API: cluster/cluster_id
3. If cluster-id is not found in #1 and #2, generate a new UUID
4. Set the cluster_id in cluster pillar using pillar_set API: cluster/cluster_id
5. Make an entry of the set cluster_id to each srvnode entry in cluster pillar

Reply:
@ypise addressed the sequence of steps. 
although, one minor clarification needed here. If cluster_id is not found steps 2 and 3, should we not break the code flow saying "bootstrap is not done, hence cluster_id cannot be set" ? Since we're generating cluster_uuid in bootstrap process also..